src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
-                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]\t");
```

Comment:
To be fair, we did not have alignment with/without my change.

If it alighment bother you, we need something like http://stackoverflow.com/questions/2485963/c-alignment-when-printing-cout